internal/controlplane/handlers_oauth_test.go

```diff
@@ -497,6 +504,7 @@ func TestProviderCallback(t *testing.T) {
 		},
 		code: 403,
 		err:  "The provided login token was associated with a different user.\n",
+		buildStubs: nil,
```

Comment:
This should be a no-op function. Setting it to nil causes a nil pointer dereference error.